### ISSUE TITLE
Variable font family name is ... "Extralight"

### ISSUE THREAD

**Issue by alerque** [theleagueof/league-spartan#28]
Similar to https://github.com/theleagueof/raleway/issues/17 the variable font builds seem to use the first named instance as the family name, which makes this project's VF *League Spartan Extralight*. Not so smooth for something with a weight axis that goes to 900.

Having poked at his a little I don't think it's even a bug in Fontship, I think it goes all the way up the tree to `fontmake`. But wherever the bug is and however it gets fixed a new release should be done for this font when this is taken care of.

**Comment by sursly** [comment]
Yeah, this is unfortunate but common. A workaround would be to insert the regular instance as a master, and make that the font origin. Not ideal to make a separate master just to account for this. I thought running **gftools fix-vf-meta** in the build was supposed to fix this, but clearly not.

**Comment by davelab6** [comment]
This shouldn't appear in the **family** name. I guess this issue can be closed?
